var/spack/repos/builtin/packages/libffi/package.py

```diff
@@ -32,6 +32,12 @@ class Libffi(AutotoolsPackage):
     patch("clang-powerpc-3.2.1.patch", when="@3.2.1%clang platform=linux")
     # ref.: https://github.com/libffi/libffi/pull/561
     patch("powerpc-3.3.patch", when="@3.3")
+    # Fix for ISSUE: https://github.com/libffi/libffi/issues/760
+    patch(
+        "https://github.com/libffi/libffi/commit/ce077e5565366171aa1b4438749b0922fce887a4.patch",
```

Comment:
You need to add the `?full_index=1` suffix (and update the checksum) to keep this from changing in the future.